src/providers/ExpandedPoolsProvider.tsx

```diff
@@ -0,0 +1,124 @@
+import { BasicPool, BasicPoolsContext } from "./BasicPoolsProvider"
+import { BasicToken, TokensContext } from "./TokensProvider"
+import {
+  EMPTY_LOADABLE,
+  LoadableType,
+  useLoadingState,
+} from "../utils/loadable"
+import React, { ReactElement, useContext, useEffect } from "react"
+
+export type ExpandedPool = Omit<
```

Comment:
For this, you're omitting the `"tokens" | "underlyingTokens" | "lpToken"` keys and readding it after. Could we potentially just overwrite here?

Reply:
I'm not sure! Makes sense that we could though. I'll give it a shot.

Reply:
nope didn't work